web/src/components/atomic/Icon.vue

```diff
@@ -22,6 +22,8 @@
   <i-carbon-in-progress v-else-if="name === 'status-running' || name === 'status-started'" class="h-6 w-6" />
   <i-bi-slash-circle-fill v-else-if="name === 'status-skipped'" class="h-6 w-6" />
   <i-bi-check-circle-fill v-else-if="name === 'status-success'" class="h-6 w-6" />
+  <i-bi-exclamation-triangle-fill v-else-if="name === 'attention'" class="h-5 w-5" />
+  <i-bi-exclamation-triangle v-else-if="name === 'warning'" class="h-5 w-5" />
```

Comment:
What's the difference between those two icons?

Reply:
One is filled one is not.

The filled one is used for the linter errors page/tab, the other one replaces the emoji used in the warning hint.

Reply:
This one.

After:
![image](https://github.com/woodpecker-ci/woodpecker/assets/3391958/50ab32fd-d4f8-4fa6-bbd3-b7961396a2df)

Before:
![image](https://github.com/woodpecker-ci/woodpecker/assets/3391958/e979b788-542b-40f2-b73d-5667244695d4)

I don't see a good way to use the same icon for both cases, but let me know if you would like to change something and I can do so.